### ISSUE TITLE
[4.x]: Showing wrong Database driver when using MariaDB

### ISSUE THREAD

**Issue by LeoWie93** [craftcms/cms#12827]
### Description
Craft is mixing the DB Version and Driver under "Systeminformation"

### Steps to reproduce

1. use Craft 3.x or 4.x on a Server with MariaDB installed

### Expected behavior
It should show:
- `Database driver & version: MariaDB 10.5.18`

### Actual behavior
![image](https://user-images.githubusercontent.com/29427760/223659435-ab781df0-e108-4897-8c0b-3a4cb3a95d23.png)

It is in not breaking the functionality of Craft. But it lead to confusions and is more of a Quality of Live issue

### Craft CMS version

Craft Pro 4.3.8

### PHP version

8.1.16

### Operating system and version

Linux 5.10.0-21-amd64

### Database type and version

MariaDB 10.5.18

### Image driver and version

GD 8.1.16

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
Make sure `yiisoft/yii2-shell` is required in `composer.json`, and then run the following command:

```sh
php craft shell
```

Then within the interactive shell, execute the following PHP code:

```php
Craft::$app->db->getSchema()->getServerVersion()
```

What does that output?

**Comment by LeoWie93** [comment]
@brandonkelly  it gives the following output:
![image](https://user-images.githubusercontent.com/29427760/224058211-a635f7a4-5420-4773-a34d-df44bed4a65f.png)

**Comment by brandonkelly** [comment]
Thanks! Just released Craft 3.8.1 and 4.4.1 with a fix for that.
